internal/build/job/maker.go

```diff
@@ -16,19 +16,43 @@ import (
 //go:generate mockgen -source=maker.go -package=job -destination=mock_maker.go
 
 type Maker interface {
```

Comment:
Maker interface is not implemented by 2 entities. Maybe we can move its definition to a manager file?

Reply:
good point, I have separated it out into its own maker.go file, and renamed the maker instance (that implements Maker) to builder because it does the build. So now we have signer.go and builder.go that implement the Maker interface. Hopefully that is both clearer and better structured than before.